tmol/tests/optimization/test_scorefunction_minimization.py

```diff
@@ -18,7 +18,8 @@ def test_minimize_w_pose_and_sfxn_smoke(rts_ubq_res, default_database, torch_dev
     pose_stack5 = PoseStackBuilder.from_poses([pose_stack1] * 5, torch_device)
 
     sfxn = ScoreFunction(default_database, torch_device)
-    sfxn.set_weight(ScoreType.fa_lj, 1.0)
+    sfxn.set_weight(ScoreType.fa_ljatr, 1.0)
+    sfxn.set_weight(ScoreType.fa_ljrep, 0.65)
```

Comment:
ljrep is 0.65 here - should this be 0.55 like Frank's comment?

Reply:
oh! I think that's a typo, it should be 0.55